frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1352,15 +1352,19 @@ const ManageHostsPage = ({
         }`}
       >
         <span>{`${count} host${count === 1 ? "" : "s"}`}</span>
-        <Button
-          className={`${baseClass}__export-btn`}
-          onClick={onExportHostsResults}
-          variant="text-link"
-        >
-          <>
-            Export hosts <img alt="" src={DownloadIcon} />
-          </>
-        </Button>
+        {count && count > 0 ? (
```

Comment:
Curious if there is a reason to use this over `!!count`?

Reply:
Or better yet, just `count`🤦

Reply:
Yep that works in the ternary context!